scripts/evaluation_script.py

```diff
@@ -48,3 +53,71 @@
     pic_loc="output/",
     pic_name="weekly_predicted_uptake.png",
 )
+
+# evaluate time-varying MSPE #
+date_mspe = pl.DataFrame()
+
+for i in range(len(rt_pred_2023)):
+
+    mspe = get_mspe(
+        data_df = nis_usa_2023,
+        pred_df = rt_pred_2023[i],
+        var = 'daily'
+    )
+    date_mspe = pl.concat(
+        [date_mspe,mspe],
+        how = 'vertical'
+    )
+
+# plot MSPE #
+time_axis = alt.Axis(
+    format = '%Y-%m',tickCount='month'
+)
+alt.Chart(date_mspe).mark_circle(color='black').encode(
+    x=alt.X('date:T',title='Date',axis=time_axis),
+    y=alt.Y('mspe',title='MSPE')
+).save(
+    'output/MSPE.png'
+)
+
+# evaluate predicted end-of-season uptake #
+date_eos = pl.DataFrame()
+
+for i in range(len(rt_pred_2023)):
```

Comment:
again, avoid loops where possible!

Reply:
Addressed in [5f2e366](https://github.com/CDCgov/cfa-immunization-uptake-projection/pull/37/commits/5f2e366c5115d1e769b602f4aacb4d854cce534e)